_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,82 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version"
+by: Tomasz Godzik
+---
+
+**TLDR;**
```

Comment:
in the TLDR section I would suggest omitting any mention of `Unsafe`. That isn't the lede; it's a technical detail. The main things the reader wants to know are: we're dropping 8; we might drop 11 too; and when this is expected to happen.

There is already material about `Unsafe` just a few lines down, anyway.

Reply:
I'd perhaps mention it in broad strokes and leave a link to the tied issue, for whoever is interested?

Reply:
Removed it and simplified the sentence